sw/host/tests/manuf/manuf_cp_unlock_raw/src/main.rs

```diff
@@ -0,0 +1,160 @@
+        .context("failed to wait for LC to be ready")?;
+
+    // Try to claim the transitions mutex.
+    poll::poll_until(Duration::from_secs(1), Duration::from_millis(10), || {
+        jtag.write_lc_ctrl_reg(
+            &LcCtrlReg::ClaimTransitionIf,
+            u8::from(MultiBitBool8::True) as u32,
+        )
+        .context("failed to write to CLAIM_TRANSITION_IF register")?;
+
+        // Check we acquired the lock
+        let got_lock = jtag
+            .read_lc_ctrl_reg(&LcCtrlReg::ClaimTransitionIf)
+            .context("failed to read CLAIM_TRANSITION_IF register")?;
+
+        Ok(got_lock == 0x01)
+    })
+    .context("failed to claim transition mutex")?;
+
+    // Check we're currently in the `RAW` lifecycle state.
+    let state = jtag
+        .read_lc_ctrl_reg(&LcCtrlReg::LcState)
+        .context("failed to read LC_STATE register")?;
+    assert_eq!(state, u32::from(DifLcCtrlState::Raw));
+
+    // Enable the external clock.
+    jtag.write_lc_ctrl_reg(
+        &LcCtrlReg::TransitionCtrl,
+        LcCtrlTransitionCtrl::EXT_CLOCK_EN.bits(),
+    )
+    .context("failed to set TRANSITION_CTRL.EXT_CLOCK_EN bit")?;
+
+    // Check that transitions are writable (we hold the mutex, so they should be).
+    let transitions_enabled = jtag
+        .read_lc_ctrl_reg(&LcCtrlReg::TransitionRegwen)
+        .context("failed to read TRANSITION_REGWEN register")?;
+    assert_eq!(transitions_enabled, 0x01);
+
+    // Provide the `RAW_UNLOCK` token
+
+    let token_regs = [
+        &LcCtrlReg::TransitionToken0,
+        &LcCtrlReg::TransitionToken1,
+        &LcCtrlReg::TransitionToken2,
+        &LcCtrlReg::TransitionToken3,
+    ];
+    // FIXME: fill this out with the correct value.
+    let token = DifLcCtrlToken::from([
+        0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, //
+        0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f,
+    ]);
+    let token_words = token.into_register_values();
+
+    for (reg, value) in iter::zip(token_regs, token_words) {
+        jtag.write_lc_ctrl_reg(reg, value)
+            .context("failed to write word to TRANSITION_TOKEN register")?;
+    }
+
+    // Set the transition target to `TEST_UNLOCKED0`.
+    jtag.write_lc_ctrl_reg(
+        &LcCtrlReg::TransitionTarget,
+        u32::from(DifLcCtrlState::TestUnlocked0),
+    )
+    .context("failed to write to TRANSITION_TARGET register")?;
+
+    // Initiate the transition.
+    jtag.write_lc_ctrl_reg(&LcCtrlReg::TransitionCmd, LcCtrlTransitionCmd::START.bits())
+        .context("failed to write to TRANSITION_CMD register")?;
+
+    // Poll until the transition succeeds
+    lc_transition::wait_for_status(
+        &jtag,
+        Duration::from_secs(1),
+        LcCtrlStatus::TRANSITION_SUCCESSFUL,
+    )
+    .context("did not read TRANSITION_SUCCESSFUL status from lifecycle controller")?;
+
+    // Reset OpenTitan.
+    // FIXME: Testplan specifies toggling the POR pin - does this do that?
```

Comment:
yes this does

Reply:
but you could change the `false` `clear_uart_rx` param to `true`

Reply:
The `lc_transition` testutil does this now